frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts

```diff
@@ -773,6 +774,56 @@ describe('Serving Runtime List', () => {
         .should('have.text', '3');
     });
 
+    it('Successfully deletes model server', () => {
+      initIntercepts({
+        projectEnableModelMesh: true,
+        disableKServeConfig: false,
+        disableModelMeshConfig: true,
+        inferenceServices: [
+          mockInferenceServiceK8sResource({ name: 'test-inferebce', isModelMesh: true }),
+          mockInferenceServiceK8sResource({
+            name: 'ovms-testing',
+            displayName: 'OVMS ONNX',
+            isModelMesh: true,
```

Comment:
```suggestion
          mockInferenceServiceK8sResource({ name: 'test-inferebce', isModelMesh: false }),
          mockInferenceServiceK8sResource({
            name: 'ovms-testing',
            displayName: 'OVMS ONNX',
            isModelMesh: false,
```
it is a kserve model also could you please update the test description.